### PR TITLE
add multiple jdks to travis build

### PR DESCRIPTION
**Pull request by devmage**
Adds all of:

* OpenJDK 7
* Oracle JDK 7
* Oracle JDK 8

to the Travis build, to help track https://github.com/paypal/cascade/issues/80

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,4 +4,10 @@ scala:
   - 2.10.4
   - 2.11.5
 
+jdk:
+  - openjdk6
```
Comment (taylorleese) [inline .travis.yml]: Isn't jdk 6 already deprecated?

**Comment by taylorleese** [comment]
@devmage Can you do the same for Horizon?

File: .travis.yml (hunk 1 above)
Comment (devmage) [inline .travis.yml]: Whether or not, Cascade targets Java 1.7+, so this was incorrect anyway.

**Comment by devmage** [comment]
@taylorleese already on it!

**Comment by arschles** [comment]
@devmage looks like it failed at least on openjdk6 for 2.10 and 2.11

**Comment by devmage** [comment]
@arschles the JDK 6 failure is expected, the `Build.scala` targets JDK 7+

**Comment by arschles** [comment]
ok @devmage can you take it out of the matrix then. want maximum coverage here but travis doesn't support expected failures

**Comment by devmage** [comment]
@arschles I've already removed JDK 6, see my second commit. Or did you mean to remove JDK 8 as well?

**Comment by arschles** [comment]
JDK8 is failing. according to https://github.com/sbt/sbt/issues/1399 it looks like it may be expected?

**Comment by devmage** [comment]
@arschles that conversation seems to suggest that 2.10.4 was the support point, which is what this was also built against. Not sure what it means for us. Perhaps we're drawing in a dependency that was built against an older Scala version?

**Comment by arschles** [comment]
perhaps

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,12 @@ language: scala
 
 scala:
   - 2.10.4
+  - 2.11.4
```
Comment (devmage) [inline .travis.yml]: This should be reverted before merging.

**Comment by devmage** [comment]
@arschles thoughts?

**Comment by taylorleese** [comment]
:shipit:

**Comment by taylorleese** [comment]
@devmage Can you do this for Horizon as well?

**Comment by devmage** [comment]
@taylorleese lol I did, and you backed out the idea because horizon can't cross-build for whatever reason

**Comment by taylorleese** [comment]
@devmage Good point. Forgot. Thx.

**Pull request merged**